app/public/js/player/playerCtrl.js

```diff
@@ -358,4 +360,85 @@ app.controller('PlayerCtrl', function (
     });
 
 
+    var server = http.createServer(function (req, res) {
+        switch (req.url) {
+            case '/playpause':
+                if ($rootScope.isSongPlaying) {
+                    playerService.pauseSong();
+                } else {
+                    playerService.playSong();
+                }
+                break;
+            case '/prev':
+                if ($rootScope.isSongPlaying) {
+                    playerService.playPrevSong();
+                }
+                break;
+            case '/next':
+                if ($rootScope.isSongPlaying) {
+                    playerService.playNextSong();
+                }
+                break;
+        }
+        res.end(`
```

Comment:
do the same thing you did here but you will need to require the page e.g `var control =  require('control.html')` and then `res.end(control)`